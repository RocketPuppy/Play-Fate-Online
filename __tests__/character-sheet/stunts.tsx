import * as React from "react";
import * as renderer from "react-test-renderer";
import Stunts from "../../src/character-sheet/stunts";

test("renders all the pieces", () => {
  const render = renderer.create(<Stunts />);

  expect(render.toJSON()).toMatchSnapshot();
});

