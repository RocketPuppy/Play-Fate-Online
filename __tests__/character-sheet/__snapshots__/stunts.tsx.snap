// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Stunts
  </h2>
  <form>
    <div>
      <div>
        <label>
          <span>
            Name
          </span>
          <input
            defaultValue=""
            onChange={[Function]}
            placeholder="Lucky"
            type="text"
          />
        </label>
        <label>
          <span>
            Description
          </span>
          <textarea
            onChange={[Function]}
            placeholder="Once per session you may roll a single dice to resolve a test. If you don't fail, succeed with style automatically."
            value=""
          />
        </label>
        <button
          disabled={true}
          onClick={[Function]}
        >
          Add new stunt
        </button>
      </div>
    </div>
  </form>
</div>
`;
